test/UnitTest/Microsoft.AspNetCore.OData.Test/Abstraction/ODataActionSelectorTestHelper.cs

```diff
@@ -0,0 +1,151 @@
+﻿// Copyright (c) Microsoft Corporation.  All rights reserved.
+// Licensed under the MIT License.  See License.txt in the project root for license information.
+
+using System.Collections.Generic;
+using System.Reflection;
+using Microsoft.AspNetCore.Mvc.Infrastructure;
+using Microsoft.AspNet.OData.Routing;
+using Microsoft.AspNet.OData.Extensions;
+using Microsoft.AspNetCore.Routing;
+using Microsoft.AspNetCore.Mvc.Controllers;
+using Microsoft.AspNetCore.Mvc.Abstractions;
+using System.IO;
+using Microsoft.AspNetCore.Mvc.ModelBinding;
+using System.Linq;
+#if NETCOREAPP2_0
+using Microsoft.Extensions.DependencyInjection;
+    using Microsoft.Extensions.Logging;
+    using Microsoft.OData;
+    using Microsoft.AspNetCore.Mvc.Internal;
+#else
+using Moq;
+#endif
+
+
+namespace Microsoft.AspNet.OData.Test.Abstraction
+{
+    /// <summary>
+    /// Helper methods for working with ODataActionSelector tests
+    /// </summary>
+    public static class ODataActionSelectorTestHelper
+    {
+        /// <summary>
+        /// Checks whether the specified method is a suitable match
+        /// for the specified action
+        /// </summary>
+        /// <param name="action"></param>
+        /// <param name="method"></param>
```

Comment:
If you add the comments, please finish them?
Otherwise, FYI, in the Test project, it's optional to add the comments.